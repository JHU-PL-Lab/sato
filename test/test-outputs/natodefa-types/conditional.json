[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "p" ],
          "err_match_val": "input + input",
          "err_match_expected": "Boolean",
          "err_match_actual": "Integer"
        }
      ]
    ],
    "err_input_seq": [ 0, 0 ],
    "err_location": "if p then true else false",
    "err_steps": 135
  }
]
