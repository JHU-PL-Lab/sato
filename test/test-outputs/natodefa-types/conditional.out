Type errors for:
- Input sequence : [0, 0]
- Found at expr  : if p then true
else false
--------------------
* Value    : p = input + input
* Expected : Boolean
* Actual   : Integer
1 errors found. No further control flows exist.
